src/Client/components/ques/problem-solving-ques.tsx

```
@@ -1,114 +1,102 @@
+        onRefresh={onRefreshTestCase}
+        onInputChange={(field, value) => onTestCaseChange(index, field, value)}
+      />
+    ))}
+    <div className="w-full flex justify-between">
+      <div>
+        <Input
+          className="w-32"
+          type="number"
+          label="Points"
+          value={problem.points.toString()}
+          onChange={(e) => onPointsChange(parseInt(e.target.value))}
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)